### PR TITLE
Add prefix for `add` operation

### LINKED ISSUE #139: Add prefix for add and remove operations
In general it is [possible](https://helm.sh/docs/howto/chart_repository_sync_example/#generate-an-updated-indexyaml) to generate index for local directory. So it can be necessary to pass prefix for `index.yaml` file to which some changes should be applied (e.g. add charts or remove them).

- [ ] prefix for add
- [x] prefix for remove

### PR DESCRIPTION
**Pull request by genryxy**
Closes #139 
Added prefix for `add` operation. Added negative tests for keys with incorrect prefix

### CODE REVIEW

**Comment by genryxy** [comment]
@g4s8 could you please take a look

**Comment by genryxy** [comment]
@g4s8 could you please check this PR

File: src/main/java/com/artipie/helm/Helm.java [hunk 1]
```diff
@@ -62,9 +62,10 @@
      * passed charts. In case of existence info about them in index
      * file an exception would be thrown.
      * @param charts Keys for charts which should be added to index file
+     * @param prefix Path to index file
```
Comment (g4s8) [inline src/main/java/com/artipie/helm/Helm.java:65]: The name of the variable seems to be wrong, I'd call it `index` or similar

File: src/main/java/com/artipie/helm/Helm.java [hunk 2]
```diff
@@ -62,9 +62,10 @@
...
      * @return Result of completion
      */
-    CompletionStage<Void> add(Collection<Key> charts);
+    CompletionStage<Void> add(Collection<Key> charts, Key prefix);
```
Comment (g4s8) [inline src/main/java/com/artipie/helm/Helm.java:68]: I'd recommend introducing a `CHANGELOG` file in repo root to track all important changes here to properly handle new versions names. We can maintain it as follows: starts block with version placeholder, e.g. `__NEXT__`, then describe changes with some prefixes:
 - fix - for bug fixes
 - new - for new functionality
 - break - for breaking changes

e.g.:
```
__NEXT__
break: #141 - Helm SDK API changed, added Key parameter
```
This way we'll handle updates correctly by reviewing last changes and updating the `__NEXT__` placeholder with major bump if needed.

**Review by g4s8 — CHANGES_REQUESTED** [review]
@genryxy please check my comments

File: src/main/java/com/artipie/helm/Helm.java (hunk 2 above)
Comment (genryxy) [reply src/main/java/com/artipie/helm/Helm.java]: @g4s8 okay, got it. I'll create this file. By the way, Helm SDK API has not been yet released

**Comment by genryxy** [comment]
@g4s8 thanks, renamed

**Pull request merged**